drivers/storage/portworx/component/telemetry.go

```diff
@@ -135,6 +135,9 @@ func (t *telemetry) createConfigMap(
         "product_name": "portworx",
         "appliance_id_path": "/etc/pwx/cluster_uuid"
       },
+      "subscription": {
+        "use_appliance_id": "true",
```

Comment:
Will have to remove the comma at the end. This will fail to parse in json.

Reply:
good catch. Fixed.